code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -5061,23 +5087,26 @@
 	name = "Lemon Juice"
 	id = LEMONJUICE
 	description = "This juice is VERY sour."
-	color = "#C6BB6E" //rgb: 198, 187, 110
+	color = "#FFFF00" //rgb: 255, 255, 000
```

Comment:
PISS COLOR

Reply:
![image](https://user-images.githubusercontent.com/23635874/94618546-55e06600-0279-11eb-9319-0e52b0c8243b.png)
this is your lemon juice for the day, say something nice about her!

Reply:
There is probably something wrong if your piss is #FFFF00

Reply:
NOT MINE I DRINK LOTS OF WATER.

But there's also something **very** wrong if your lemon juice is #FFFF00

Reply:
okay it's #FFF690 now